libs/plutus-preprocessor/plutus-preprocessor.cabal

```
@@ -42,7 +42,7 @@ library
     bytestring,
     cardano-ledger-binary:testlib,
     cardano-ledger-core,
-    plutus-ledger-api ^>=1.37,
+    plutus-ledger-api ^>=1.42,
```

Per my https://github.com/IntersectMBO/cardano-ledger/pull/4923#issuecomment-2726057036
```suggestion
    plutus-ledger-api >=1.43,
```